app/src/main/java/org/mozilla/fenix/components/metrics/MetricController.kt

```diff
@@ -114,6 +115,16 @@ internal class ReleaseMetricController(
         Component.FEATURE_PROMPTS to LoginDialogFacts.Items.SAVE -> {
             LoginDialog.saved.record(NoExtras())
         }
+        Component.BROWSER_TOOLBAR to ToolbarFacts.Items.MENU -> {
+            metadata?.get("customTab")?.let { CustomTab.menu.record(mozilla.telemetry.glean.private.NoExtras()) }
+                ?: track(Event.ToolbarMenuShown)
```

Comment:
Should we also unwrap `Event.ToolbarMenuShown` at the same time?

Reply:
The unwrapping of `Event.ToolbarMenuShown` should be handled in [this PR](https://github.com/mozilla-mobile/fenix/pull/24483/files#diff-b7d8c440fde253f247963ad4f07a0fa0f4ac5f68258ede13ab6667b3abb7301fR120).

Reply:
I see.  Thanks for the clarification.